holodeck/environments.py

```diff
@@ -56,8 +56,9 @@ def __init__(self, agent_name, agent_type, sensors=None):
 class HolodeckEnvironment(object):
     """The high level interface for interacting with a Holodeck world"""
 
-    def __init__(self, agent_definitions, binary_path=None, task_key=None, height=512, width=512,
-                 start_world=True, uuid="", gl_version=4, verbose=False):
+    def __init__(self, agent_definitions, window_height=512, window_width=512, camera_height=256, camera_width=256,
+                 binary_path=None, task_key=None,start_world=True, uuid="", gl_version=4):
```

Comment:
Nits: Should be a space between `task_key=None,start_world=True`. Quickly run reformat file in pycharm to make sure everything is formatted correctly

Reply:
Also, verbose flag has been removed